packages/block-library/src/latest-posts/edit.js

```diff
@@ -424,17 +445,18 @@ class LatestPostsEdit extends Component {
 
 						return (
 							<li key={ i }>
-								{ displayFeaturedImage && (
+								{ renderFeaturedImage && (
 									<div className={ imageClasses }>
-										{ imageSourceUrl && (
-											<img
-												src={ imageSourceUrl }
-												alt=""
-												style={ {
-													maxWidth: featuredImageSizeWidth,
-													maxHeight: featuredImageSizeHeight,
-												} }
-											/>
+										{ addLinkToFeaturedImage ? (
+											<a
+												href={ post.link }
+												target="_blank"
```

Comment:
should this target be blank by default?

Reply:
The truth is that it makes sense in the editor to be `blank`, so as not to click and leave the editing process. The same happens (and was there before) for `title and excerpt` here. This component definitely needs refactoring and polishing but for this PR and the small addition feels okay, no?

There will be more follow up PRs for handling several stuff and most importantly is the addition of selecting the `target` and `rel` of these links. Because in front-end no `target` or `rel` is set, so it gets the default values.